01-getting-started/getting-started.Rmd

```diff
@@ -37,10 +37,10 @@ Each analysis is self-contained and provides information with how to obtain the
 We encourage you to download the [`.Rmd`](http://rmarkdown.rstudio.com) and follow the "getting started" section in the example before diving into our analysis examples.
 
 **Each analysis contains:**   
-- A README that introduces you to the analyses, concepts, requirements, and workflows for that module.
-- An R Notebook which consists of:
-     - An R markdown (`.Rmd`) file(s) that you can use in RStudio to run the analysis and contains it's own "getting started" section which describes how to download the example dataset from refine.bio. 
-     - An nb.html file that is the resulting output of the Rmd file rendered as an HTML file.
+- A README that introduces you to the analyses, concepts, requirements, and workflows for that module.  
```

Comment:
This will fix the rendering. You need a blank line above bullet lists.
```suggestion

- A README that introduces you to the analyses, concepts, requirements, and workflows for that module.  
```